app/views/items/index.html.erb

```diff
@@ -91,11 +91,9 @@
             <% @items.each do |item| %>
               <%= link_to item_path(item), class: "text-decoration-none" do%>
                 <div class="card-product <%= 'gray' if (item.donated? || item.reserved?) %>">
-                  <% if item.photos.attached? %>
-                    <%= cl_image_tag item.photos.first.key, width: "100%", height: 400 %>
-                  <% else %>
-                    <img src="https://images.unsplash.com/photo-1554136383-fa88b2d86aaf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=928&q=80" alt="" width = 100% height = 400>
-                  <% end %>
+                  <div class="image">
+                    <%= cl_image_tag item.photos.first.key, class: "img" %>
+                  </div>
 
```

Comment:
each item must have a photo attached (through cloudinary), so the `else` statement is removed.